### PR TITLE
remove computed attributes

### PR DESCRIPTION
**Pull request by lamont-granquist**
they don't work.

### CODE REVIEW

**Comment by tas50** [comment]
What if someone wants to host the source file on a different url?

**Comment by lamont-granquist** [comment]
`node['erlang']['source']['url']` still works just fine.

File: recipes/source.rb [hunk 1]
```diff
@@ -37,21 +37,27 @@
   end
 end
 
+erlang_version     = node['erlang']['source']['version']
+erlang_url         = node['erlang']['source']['url'] || "http://erlang.org/download/otp_src_#{erlang_version}.tar.gz"
```
Comment (lamont-granquist) [inline recipes/source.rb]: ^ that is functionally identical to the previous code, only working.

ruby variables >> attributes.

**Comment by tas50** [comment]
I need to learn to read apparently

**Pull request merged**